api/backend/ecs/load_balancer_manager.go

```diff
@@ -349,6 +364,15 @@ func (e *ECSLoadBalancerManager) UpdateLoadBalancerHealthCheck(loadBalancerID st
 	return e.GetLoadBalancer(loadBalancerID)
 }
 
+func (e *ECSLoadBalancerManager) UpdateLoadBalancerIdleTimeout(loadBalancerID string, idleTimeout int) (*models.LoadBalancer, error) {
```

Comment:
Update should only return errors; adding `return e.GetLoadBalancer(loadBalancerID)` is coupling this function to `GetLoadBalancer`, which will make this much harder to test

Reply:
Why does `UpdateLoadBalancerHealthCheck` use this pattern in that case? I modeled this behavior off of the `healthcheck` subcommand.

Reply:
ah I see - was thinking about `api-refactor` patterns instead of the ones in `develop` - disregard!